### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by Hoverbear**
##### Description

Routine dep bump.

##### Checklist

- [ ] Formatted with `cargo fmt`
- [ ] Built with `nix build`
- [ ] Ran flake checks with `nix flake check`
- [ ] Added or updated relevant tests (leave unchecked if not applicable)
- [ ] Added or updated relevant documentation (leave unchecked if not applicable)
- [ ] Linked to related issues (leave unchecked if not applicable)

##### Validating with `install.determinate.systems`

If a maintainer has added the `upload to s3` label to this PR, it will become available for installation via `install.determinate.systems`:

```shell
curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix/pr/$PR_NUMBER | sh -s -- install
```

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -136,6 +136,7 @@
               toolchain
               rust-analyzer
               cargo-outdated
+              cacert
```
Comment (cole-h) [inline flake.nix:139]: Hm? Why is this needed?

Comment (Hoverbear) [reply flake.nix:139]: Running `cargo outdated` gave me an error until I added this. :woman_shrugging:

**Review by cole-h — APPROVED** [review]
